base/src/main/java/com/smartdevicelink/managers/screen/choiceset/PresentKeyboardOperation.java

```diff
@@ -284,6 +284,8 @@ public void onNotified(RPCNotification notification) {
                 if (onKeyboard.getEvent().equals(KeyboardEvent.ENTRY_VOICE) || onKeyboard.getEvent().equals(KeyboardEvent.ENTRY_SUBMITTED)) {
                     // Submit Voice or Text
                     keyboardListener.onUserDidSubmitInput(onKeyboard.getData(), onKeyboard.getEvent());
+                } else if (onKeyboard.getEvent().equals(KeyboardEvent.INPUT_KEY_MASK_ENABLED) || onKeyboard.getEvent().equals(KeyboardEvent.INPUT_KEY_MASK_DISABLED)) {
+                    keyboardListener.onMaskHasChanged(onKeyboard.getEvent());
```

Comment:
@kostyaBoss Do the unit tests in `PresentKeyboardOperationTests.java` need to be updated?

Reply:
@santhanamk there were no tests covering the mask previously so I didn't add some new ones. To my mind, the PresentChoiceSetOperationTests are more about thread-safe testing, but we are not introducing some major changes at this point, so I don't think we should push the changes. Please, let me know if you still think we need to update them